pkg/agent/openflow/pipeline.go

```diff
@@ -917,7 +917,7 @@ func (c *client) serviceLBBypassFlows(ipProtocol binding.Protocol) []binding.Flo
 // l2ForwardCalcFlow generates the flow that matches dst MAC and loads ofPort to reg.
 func (c *client) l2ForwardCalcFlow(dstMAC net.HardwareAddr, ofPort uint32, skipIngressRules bool, category cookie.Category) binding.Flow {
 	l2FwdCalcTable := c.pipeline[l2ForwardingCalcTable]
-	nextTable := l2FwdCalcTable.GetNext()
+	nextTable := IngressMetricTable
```

Comment:
Could we just set IngressMetricTable as the next table when constructing the pipeline? otherwise it would still be confusing that why it's linked to conntrackCommitTable bu default but no flows actually hit it. 
It's better to add a comment above that line where you updates the next table to explain why it should go to IngressMetricTable instead of conntrackCommitTable

Reply:
updated